ballerina/Module.md

```diff
@@ -23,7 +23,3 @@ import ballerina/sql;
 import ballerinax/oracledb;
 import ballerinax/oracledb.driver as _;
 ```
-
```

Comment:
Why is this removed?

Reply:
We usually don't have the Useful links section in `Module.md`. Instead we have them in `Package.md`

e.g.
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Module.md
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Package.md